sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainerProactiveInitConfig.java

```diff
@@ -96,17 +96,19 @@ public String toString() {
             return "";
         }
 
-        return
-            String.format(
-                "%s(%d)",
+        // containers -> the containers part of the connection warm up
+        // pcrc -> the first k regions from the preferredRegions to which connections are warmed up
+        // awd -> duration within which aggressive connection warm up happens
+        return String.format(
+                "(containers:%s)(pcrc:%d)(awd:%s)",
                 cosmosContainerIdentities
                     .stream()
                     .map(ci -> String.join(
                         ".",
-                        containerIdAccessor.getDatabaseName(ci),
-                        containerIdAccessor.getContainerName(ci)))
-                    .collect(Collectors.joining(",")),
-                numProactiveConnectionRegions);
+                        containerIdAccessor.getContainerLink(ci)))
```

Comment:
why are we using containerLink and not containerName?

Reply:
Here `containerLink` refers to `/dbs/{databaseName}/colls/{containerName}` where `databaseName` and `containerName` are supplied by the client application.